workshops/3d_models_with_zdog/README.md

```diff
@@ -0,0 +1,257 @@
+```
+<script src=“https://unpkg.com/zdog@1/dist/zdog.dist.min.js”></script>
+```
+At the end, your code should look kind of like this:
+```html
+<body>
+    <canvas class="model"></canvas>
+    <script src="https://unpkg.com/zdog@1/dist/zdog.dist.min.js"></script>
+    <script src="script.js"></script>
+ </body>
+```
+*Note: It’s very important to keep this order to make sure everything works perfectly*
+
+## JS part
+Now that we have our html file ready to go, we gotta work on our javascript file. 
+![Cool gif](https://cloud-p49mi1lgl.vercel.app/0tumblr_e49d74c805eec46704d22c1da59ecded_cc93a056_500.gif)
+
+### Setting up the canvas
+
+We are going to start with the funny part. First, let’s create a main variable and let’s give it a name, I will name it as “ws”
+
+```javascript
+const ws = new Zdog.Illustration({
+  element: ‘.model’,
+  resize: ‘fullscreen’
+});
+```
+Let's break this down:
+
+1. **Illustration** is the top-level class that handles dealing with the <canvas> element, holding all the shapes in the scene, and displaying those shapes in the element.
+2. **element** we use this element to match the render with the canvas tag
+3. **resize** is used to modify the size in which the model will be render, in this case, the 3d model will render in the whole screen. If you want to, you can remove this element.
+
+At this point nothing shows up yet, so let’s create our first 3d model.
+
+### Creating the model
+
+Now, we’re gonna create a shape, for this workshop I want to make a simple cube, but you can create whatever you want to. Here is a list of [shapes](https://zzz.dog/shapes) you can create with Zdog 
+
+*Note: Every shape has its own properties or elements, you can check them out [here](https://zzz.dog/shapes)*
+
+Let’s add the following code to our js file:
+
+```javascript
+    const cube = new Zdog.Box({
+      addTo: ws,
+      width: 100,
+      height: 100,
+      depth: 100,
+      stroke: false,
+      leftFace: "#da0",
+      rightFace: "#e62",
+      topFace: '#ed0',
+      bottomFace: '#636'
+    })
+```
+Explanation:
+1. Box is a shape class, you can replace this for the shape you want to use…
+2. We made the 3d model (cube) a child to the main Zdog Illustration (ws) using the addTo element. This element must be there, otherwise the 3d model will not render.
+3. The width, height and depth elements can either stretch or shrink the shape of your box:
+- Width: Sets the width of the cube
+- Height: Sets the height of the cube
+- Depth: Sets the depth of the cube. If the value is 0, the cube will render as a 2d square; so make sure to give it a value.
+4. The Stroke element gives the 3d model a stroke, it works as an external layer that you can use to give your 3d model a rounded look. Play around with it!
+5. leftFace, rightFace, topFace & bottomFace elements give a color to each face of the 3d model, try using different colors for each face, so that you can appreciate way better the animations you make.
```

Comment:
```suggestion
Explanation:
1. Box is a shape class, you can replace this for the shape you want to use…
2. We made the 3d model (cube) a child to the main Zdog Illustration (ws) using the addTo element. This element must be there, otherwise the 3d model will not render.
3. The width, height, and depth elements can either stretch or shrink the shape of your box:
- Width: Sets the width of the cube
- Height: Sets the height of the cube
- Depth: Sets the depth of the cube. If the value is 0, the cube will render as a 2d square; so make sure to give it a value.
4. The Stroke element gives the 3d model a stroke, it works as an external layer that you can use to give your 3d model a rounded look. Play around with it!
5. leftFace, rightFace, topFace & bottomFace elements give color to each face of the 3d model, try using different colors for each face, so that you can appreciate way better the animations you make.
```